Source/Scene/Model.js

```diff
@@ -612,10 +656,31 @@ define([
         }
     });
 
+    function getBasePath(url) {
+        var basePath = '';
+        var i = url.lastIndexOf('/');
+        if (i !== -1) {
+            basePath = url.substring(0, i + 1);
+        }
+
+        return basePath;
+    }
+
+    function getAbsoluteURL(url) {
```

Comment:
I wrote up #2673 to move this to a helper function at some point and clean up the several other places we have similar code.